projects/aws/bottlerocket-bootstrap/buildspecs/batch-build.yml

```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 
 # This file was generated by running `make generate`. DO NOT EDIT
-
```

Comment:
unintentional?

Reply:
no that was added when i ran `make generate`